lib/db.js

```diff
@@ -750,7 +755,7 @@ Db.prototype.ensureIndex = function(name, fieldOrSpec, options, callback) {
     options,
     callback
   ]);
-};
+}, 'Db.ensureIndex is deprecated since version 2.0');
```

Comment:
I think you should phrase this and the deprecation message for Db.eval similarly.